src/components/Experiment/Footer.less

```diff
@@ -1,12 +1,14 @@
 @import '../../../node_modules/bootstrap/less/variables.less';
 
 .ex-footer {
-    border-top: @navbar-default-border;
-    margin-bottom: 20px;
+    background: #2c3132;
```

Comment:
I found many repeated colors.
- Do you think would be a good idea to have a `palette.less` stylesheet to define each color, and then reuse its values?
- We have 18 different colors. Do you think that the palette could be compressed more considering the most similar?

Here are the colors I found in our app:
(duplicated color, means it is repeated in different places in our stylesheets)

![image](https://user-images.githubusercontent.com/2437584/35683569-ac0e0c02-0764-11e8-8e03-bb0f945bb569.png)

in raw:

```
#ffffff x4
#d8dcdc x2  
rgba(0, 0, 0, 0.2)
#cccccc x2
#a0a2a2 x2
#979a9a
#818a8a x2
#808b8c x3
#444444
#2c3132 x2
#191b1c
#0c0d0e x2
#00f3ff x2
#66ffba x2
#f37b6f x2
#006166
#004c2a
#61312c
```

Reply:
Agreed 👍